Realm/RLMObjectStore.mm

```diff
@@ -28,6 +28,7 @@
 #import "RLMRealm_Private.hpp"
 #import "RLMSchema_Private.h"
 #import "RLMUtil.hpp"
+#import "RLMSwiftSupport.h"
```

Comment:
I don't really care what the ordering is as long as it's consistent between files, and currently they're all sorted alphabetically (except for the header for the current implementation file, which is always first).